google-cloud-storage/src/test/java/com/google/cloud/storage/ITMultipartUploadHttpRequestManagerTest.java

```diff
@@ -520,11 +520,11 @@ public void sendAbortMultipartUploadRequest_error() throws Exception {
               .uploadId("test-upload-id")
               .build();
 
-          assertThrows(
-              HttpResponseException.class,
-              () ->
-                  multipartUploadHttpRequestManager.sendAbortMultipartUploadRequest(
-                      endpoint, request, httpStorageOptions));
+      assertThrows(
+          HttpResponseException.class,
+          () ->
+              multipartUploadHttpRequestManager.sendAbortMultipartUploadRequest(
+                  endpoint, request, httpStorageOptions));
```

Comment:
Not related to this change, please revert here.

Reply:
WIP changes. It was a draft PR.